docs/guides/software-lifecycle/security/container-security/README.mdx

```diff
@@ -0,0 +1,130 @@
+import CodeBlock from '@theme/CodeBlock';
+import PreCommitConfigSource from '!!raw-loader!./.pre-commit-config.yml';
+import DependabotSource from '!!raw-loader!./dependabot.yml';
+
+# Container Security
+
+<pre align="center">Comprehensive guide to scanning container images for security vulnerabilities using pre-commit hooks and automated repository scanning tools.</pre>
+
+![banner-image](/img/container-security-screen.png)
+
+## Introduction
+
+**Background**: Container security is crucial for ensuring that your applications and services run in a secure environment. Containers encapsulate software and dependencies, providing consistency across environments. However, they also carry the risk of vulnerabilities within the base images or dependencies. This guide will help you establish a secure environment by implementing proactive vulnerability scanning. By using pre-commit hooks and repository-level automated scanning, you'll be able to identify security issues early, before they impact production.
+
+**Use Cases**:
+- Running vulnerability scans for all containers at the development stage
+- Ensuring that base images used in CI/CD pipelines are free from known vulnerabilities
+- Automating container vulnerability scans in repositories hosting container images
+
+---
+
+## Prerequisites
+**Software:**
+- Docker containers
+- `pre-commit` framework
+- Docker Hub or GitHub Dependabot
+
+**Skills:**
+- Basic understanding of Git hooks and Docker commands
+- Familiarity with YAML files for pre-commit configuration
+
+---
+
+## Quick Start
+
+**Run a local scan of your container's repository (folder containing the Dockerfile)**
+
+```
+docker scout cves fs://.
+```
+
+**⬇️ [.pre-commit-config.yml](.pre-commit-config.yml)**
+
+Download the file above to access the pre-commit configuration file, which includes an example hook for Docker Scout vulnerability scanning. The file should be placed within your local Git repository. You'll want to ensure you have the [pre-commit](https://pre-commit.com) framework installed. 
```

Comment:
The first comment [here](https://www.reddit.com/r/selfhosted/comments/12mvpq2/open_source_container_scanning_tool_to_find/) has a number of OSS alternatives for container scanning. I'm not saying this is the wrong way to go, it is likely right, just that the world is a bit bigger. There's also [OpenSCAP which offers the OSS Atomic tool](https://www.open-scap.org/tools/). [Podman - Trivy](https://aquasecurity.github.io/trivy/v0.17.2/scanning/advanced/podman/) -- Trivy is quite generic. [CrowdStrike Container Image Scan](https://github.com/CrowdStrike/container-image-scan).

Reply:
Using grype now, so this is outdated.